server_integration_test.go

```diff
@@ -649,6 +649,53 @@ func TestServerPut(t *testing.T) {
 	}
 }
 
+func TestServerResume(t *testing.T) {
+	listenerGo, hostGo, portGo := testServer(t, GolangSFTP, READONLY)
+	defer listenerGo.Close()
+
+	tmpFileLocal := "/tmp/" + randName()
```

Comment:
Any reason to not use https://golang.org/pkg/io/ioutil/#TempFile ?

Reply:
This and all the below are due to my having cut-n-pasted the TestServerPut test to make this. Many other tests have these same issues and I'm not sure if correcting them in one place is better than leaving them as is for consistency.

I think I'll leave this commit as is (after adding the documentation updates I forgot) but make a quick pass fixing these sorts of things where I see them in a separate commit. Do you think that commit would be better included in this PR or out on its own?

Reply:
Sounds good to just clean them all up in one pass together.